tests/integration/features/features_int_test.go

```diff
@@ -241,13 +188,10 @@ var _ = Describe("feature cleanup", func() {
 			})
 
 			It("should indicate successful installation in FeatureTracker", func() {
-				// given example CRD installed into env
-				name := "test-resources.openshift.io"
 				featuresHandler := feature.ClusterFeaturesHandler(dsci, func(handler *feature.FeaturesHandler) error {
-					verificationFeatureErr := feature.CreateFeature("crd-verification").
+					verificationFeatureErr := feature.CreateFeature("feature-success").
```

Comment:
Maybe rename it to `noop-feature`? But I am not entirely sure it's any better :)